android/app/src/main/java/updated/mysterium/vpn/model/filter/NodePrice.kt

```diff
@@ -0,0 +1,16 @@
+package updated.mysterium.vpn.model.filter
+
+import updated.mysterium.vpn.model.manual.connect.PriceLevel
+
+enum class NodePrice {
+    LOW, MEDIUM, HIGH;
+
+    companion object {
+
+        fun parseFromPriceLevel(priceLevel: PriceLevel) = when (priceLevel) {
```

Comment:
Just rename this method to `from(priceLevel: PriceLevel)`.

Reply:
Fix